examples/complete/fixtures.defaults.tfvars

```diff
@@ -0,0 +1,11 @@
+region = "us-east-2"
```

Comment:
There's already a `fixtures.us-east-2.tfvars`.

Let's call this file `fixtures-default-route-table.defaults.tfvars`